web/components/input/select/index.sass

```diff
@@ -51,4 +51,5 @@
       img
         width: 12.6px
         height: 10px
-
+.react-select__placeholder
+  white-space: nowrap
```

Comment:
If you do this the hover color gets cut off and the input covers the whole screen width
![image](https://user-images.githubusercontent.com/12876696/47823727-cba6c480-dd26-11e8-85f6-3be72f84b3c3.png)

Reply:
hm doesn't happen to me.. does that happen on master for u too?

![image](https://user-images.githubusercontent.com/21287547/47824596-d8c5b280-dd2a-11e8-8606-9452f36333b8.png)

Reply:
Oh now that you mention it, it also happens on master. You can see if you scroll to the right.

Reply:
Do you mind if we make a separate issue for it then? Not sure how high priority it is though

Reply:
Yeah no problem, was this code supposed to fix it or does it do something else?

Reply:
No it just prevents the placeholder text from wrapping on mobile view

Reply:
I guess I'm missing something cause I don't see the placeholder wrapping on either master or this branch. :joy: A screenshot of placeholder wrapping happening on master would be helpful, in the meantime I'm gonna approve this for the placeholder fixes